polliwog/plane/_plane_object.py

```diff
@@ -87,6 +91,48 @@ def fit_from_points(cls, points):
 
         return cls(centroid, normal)
 
+    def serialize(self, position_decimals=None, direction_decimals=None):
+        if position_decimals is None:
+            position_decimals = self.DEFAULT_POSITION_DECIMALS
+        if direction_decimals is None:
+            direction_decimals = self.DEFAULT_DIRECTION_DECIMALS
```

Comment:
I'm sure there's a reason for this, but why not assign these as the default value above?

Reply:
Yea, good question. The advantage of this is it allows you to override the default precision on an instance of Polyline. (I could be worth playing around with that to convince yourself that this approach is necesary.)